Client/cefweb/CWebApp.cpp

```diff
@@ -28,7 +28,10 @@ void CWebApp::OnRegisterCustomSchemes(CefRawPtr<CefSchemeRegistrar> registrar)
 void CWebApp::OnBeforeCommandLineProcessing(const CefString& process_type, CefRefPtr<CefCommandLine> command_line)
 {
     command_line->AppendSwitch("disable-gpu-compositing");
```

Comment:
Should also remove this and test to make sure it still works okay.

Reply:
```suggestion
```